client/src/components/NewsCard.js

```diff
@@ -0,0 +1,106 @@
+import styled from 'styled-components';
+import { ReactComponent as FavoriteIcon } from '../image/btc-icon.svg';
+
+export default function NewsCard({ articles, onToggleBookmarkNews }) {
+  return (
+    <>
+      {articles
+        .filter(
+          (article, index) =>
+            !articles.some(
+              (articleToCheck, articleToCheckIndex) =>
+                articleToCheck.title === article.title &&
+                articleToCheckIndex !== index
+            )
+        )
+        .map((article) => (
+          <p>
+            {article.urlToImage === null ||
+            article.description === 0 ||
+            article.author === null ? null : (
+              <ArticleWrapper>
+                <FavoriteButton>
+                  <FavoriteIcon
+                    alt="Bookmark Icon"
+                    onClick={() => onToggleBookmarkNews(article)}
+                  />
+                </FavoriteButton>
+                <NewsImage src={article.urlToImage} alt="Image of article" />
+                <Title>{article.title}</Title>
+                <TitleAndAutorWrapper>
+                  <Author>{article.author}</Author>
+                  <PublishedAt>
+                    {article.publishedAt.slice(0, 10)}{' '}
+                    {article.publishedAt.slice(11, 19)}
+                  </PublishedAt>{' '}
+                </TitleAndAutorWrapper>
+                <p>
+                  {article.description}
+                  <a href={article.url} alt="Link of article">
+                    {' '}
+                    Go to full article
+                  </a>
+                </p>
+              </ArticleWrapper>
+            )}
+          </p>
+        ))}
+    </>
+  );
+}
+
+const ArticleWrapper = styled.div`
+  background-color: var(--grey-darkest);
+  display: flex;
+  flex-direction: column;
+  padding: 0 0.625rem;
+  position: relative;
+
+  a {
+    text-decoration: none;
+  }
+
+  svg {
+    fill: orange;
+    height: 2.5rem;
+    stroke: var(--black);
+    stroke-width: 3px;
+    width: 2.5rem;
+  }
+`;
+
+const FavoriteButton = styled.button`
+  background-color: transparent;
+  border: none;
+  left: 25%;
```

Comment:
consider sorting it alphabetically, or put left together with position since they kinda go together :)

Reply:
fixed :)